src/client/bb-client.js

```diff
@@ -284,6 +284,11 @@ BBClient.ready = function(input, callback, errback){
 
   if (isFakeOAuthToken()) {
     accessTokenResolver = completePageReload();
+    if (window.history.replaceState
+      && BBClient.settings.replaceBrowserHistory
+      && BBClient.settings.fullSessionStorageSupport){
+      window.history.replaceState({}, "", window.location.toString().replace(window.location.search, ""));
+    }
```

Comment:
Should we put an error or warning message in the else case to let the user know the failure?

Reply:
Everything should work, but yes, the `?state=abc123` will be present in the URL if it fails this `if` logic.